test/browser/features/support/maze-config.rb

```diff
@@ -16,9 +16,15 @@ def get_test_url()
     maze_address = "#{host}:9339"
   end
 
+  if Maze.config.https
+    protocol = 'https'
+  else
+    protocol = 'http'
+  end
+
```

Comment:
Ruby has the ternary operator, although I'm not suggesting you suffer another build to change it, just do it on another branch.
```suggestion
  protocol = Maze.config.https ? 'https' : 'http'
```